lib/ldclient-rb/config.rb

```diff
@@ -77,6 +78,7 @@ def initialize(opts = {})
       @application = LaunchDarkly::Impl::Util.validate_application_info(opts[:application] || {}, @logger)
       @payload_filter_key = opts[:payload_filter_key]
       @hooks = (opts[:hooks] || []).keep_if { |hook| hook.is_a? Interfaces::Hooks::Hook }
+      @omit_anonymous_contexts = opts.has_key?(:omit_anonymous_contexts) && opts[:omit_anonymous_contexts]
```

Comment:
Should it follow same pattern of other boolean config options like on line 63?
```
opts.has_key?(:offline) ? opts[:offline] : Config.default_offline
```

Reply:
Great question - I am trying to follow the pattern in line 71, but I don't know which one is more Ruby-ish

```
@diagnostic_opt_out = opts.has_key?(:diagnostic_opt_out) && opts[:diagnostic_opt_out]
```

Reply:
I am inclined to think it probably doesn't matter much, and if it does we could clean them both up after.